### PR TITLE
Issue #7: Adding getSiteName function.

### LINKED ISSUE #7: Need BLT's getSiteName function in AcquiaDrupalEnvironmentDetector.
I'm building a stripped down version of BLT's[ site settings include detection](https://github.com/acquia/blt/blob/12.x/settings/blt.settings.php#L90) for a project that does not use BLT. I'd like to add the getSiteName function to this library if we can. I'll submit a PR. Thanks!

### PR DESCRIPTION
**Pull request by msherron**
Fix #7

### CODE REVIEW

File: src/AcquiaDrupalEnvironmentDetector.php [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 namespace Acquia\DrupalEnvironmentDetector;
 
+use Symfony\Component\Console\Input\ArgvInput;
```
Comment (danepowell) [inline src/AcquiaDrupalEnvironmentDetector.php:5]: You don't need this, given that you excluded some parts of the BLT method.

Comment (msherron) [reply src/AcquiaDrupalEnvironmentDetector.php]: 👍

File: src/AcquiaDrupalEnvironmentDetector.php [hunk 2]
```diff
@@ -193,6 +193,28 @@ public static function getAcsfDbName() {
     return isset($GLOBALS['gardens_site_settings']) && self::isAcsfEnv() ? $GLOBALS['gardens_site_settings']['conf']['acsf_db_name'] : NULL;
   }
 
+  /**
+   * Get a standardized site / db name.
+   *
+   * On ACE or simple multisite installs, this is the site directory under
+   * 'docroot/sites'.
```
Comment (wimleers) [inline src/AcquiaDrupalEnvironmentDetector.php:200]: Does this mean it will return `default` on most single site installs?

Comment (msherron) [reply src/AcquiaDrupalEnvironmentDetector.php:200]: Yes. This is how I'm leveraging it to find the automatically generated settings files within ACE:

`/**
 * Acquia Cloud automatically generates a settings file that contains the
 * database credentials for a given Drupal application, along with some other
 * required plumbing. The code below uses a plugin to load the correct file
 * based on the current product, site, and environment.
 */
$site_name = AcquiaDrupalEnvironmentDetector::getSiteName($site_path);
if (AcquiaDrupalEnvironmentDetector::isAhEnv()) {
  $ah_group = AcquiaDrupalEnvironmentDetector::getAhGroup();
  if (!AcquiaDrupalEnvironmentDetector::isAcsfEnv()) {
    if ($site_name == 'default') {
      require "/var/www/site-php/$ah_group/$ah_group-settings.inc";
    } else {
      // Acquia Cloud does not support periods in db names.
      $safe_site_name = str_replace('.', '_', $site_name);
      require "/var/www/site-php/$ah_group/$safe_site_name-settings.inc";
    }
  }
}`

**Pull request merged**